scylla/README.md

```diff
@@ -245,4 +290,4 @@ of 3 nodes of i3.4xlarge (16 vCPU per node) and target of 120000 is:
 
 - `scylla.lwt`
   - Use LWT for operations
-  - Default is false.
+  - Default is true.
```

Comment:
revert
default is false:
https://github.com/brianfrankcooper/YCSB/blob/master/scylla/src/main/java/site/ycsb/db/scylla/ScyllaCQLClient.java#L65

Reply:
(and I'd keep it false to be in sync with cassandra)

Reply:
Thx! did this by mistake.